src/libaktualizr/primary/eventsinterpreter.cc

```diff
@@ -18,29 +22,46 @@ EventsInterpreter::~EventsInterpreter() {
 
 void EventsInterpreter::interpret() { thread = std::thread(std::bind(&EventsInterpreter::run, this)); }
 
-std::shared_ptr<command::BaseCommand> EventsInterpreter::handle_cycle(event::BaseEvent &event, bool forever) {
+std::shared_ptr<command::BaseCommand> EventsInterpreter::handle_cycle(event::BaseEvent &event,
```

Comment:
Is something preventing us from defining a new method? I kept `once` and `full` because they are really similar but `none` seems sufficiently different and simple:

```
std::shared_ptr<command::BaseCommand> EventsInterpreter::handle_none(event::BaseEvent &event) {
  if (event.variant == "SendDeviceDataComplete") {
    return std::make_shared<command::FetchMeta>();
  } else if (event.variant == "FetchMetaComplete") {
    return std::make_shared<command::CheckUpdates>();
  }
  return nullptr;
}
```

Reply:
Done. Good call.